sdk/appcenter-distribute/src/main/java/com/microsoft/appcenter/distribute/Distribute.java

```diff
@@ -1648,6 +1648,12 @@ public void onClick(DialogInterface dialog, int which) {
      */
     private synchronized void goToUnknownAppsSettings(ReleaseDetails releaseDetails) {
         Intent intent;
+        if (mForegroundActivity == null) {
+
+            /* We should not open settings screen if app is in background now. */
```

Comment:
Can we add more details about why exactly we can't open the settings screen in a background mode?
Or remove this line at all, as the message in `AppCenterLog.warn` tells us the same thing.